trin-execution/src/constants.rs

```diff
@@ -0,0 +1,2 @@
+// The location in the database which describes the current execution position.
+pub const CURRENT_EXECUTION_POSITION: &[u8; 26] = b"current_execution_position";
```

Comment:
nit: I think this should be better suited in `trin-execution/src/storage/db_key.rs` or something like that. I also don't think that `CURRENT_` is needed in the name, as we are storing only one `EXECUTION_POSITION`.